testing/spectest/minimal/bellatrix/forkchoice/forkchoice_test.go

```diff
@@ -9,11 +9,16 @@ import (
 )
 
 func TestMinimal_Bellatrix_Forkchoice(t *testing.T) {
+	resetCfg := features.InitWithReset(&features.Flags{
+		PullTips: true,
```

Comment:
Do we care about permutation where PullTips is false?

Reply:
Oh no, this is not going to be there when we merge, this is only here to pass testvectors will be gone when I manage to resolve the compilation problems that I don't know where they are coming from